server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_1__add_integration_git.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS `integration_git_repository` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `create_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `update_time` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `creator_id` bigint(20) NOT NULL COMMENT 'creator user id, references iam_user.id',
+  `organization_id` bigint(20) NOT NULL COMMENT 'organization id, references iam_organization.id',
+  `project_id` bigint(20) NOT NULL COMMENT 'project id, references collaboration_project.id',
+  `description` varchar(512) DEFAULT NULL COMMENT 'description',
+  `name` varchar(512) NOT NULL COMMENT 'repository name',
+  `provider_type` varchar(32) NOT NULL COMMENT 'git provider type',
+  `provider_url` varchar(128) NOT NULL COMMENT 'git provider url',
+  `ssh_url` varchar(512) NOT NULL COMMENT 'ssh clone url, like git@github.com:xxx/xxx.git',
+  `clone_url` varchar(512) NOT NULL COMMENT 'https clone url, like https://github.com/xxx/xxx.git',
+  `email` varchar(128) NOT NULL COMMENT 'user email for commit',
+  `token` varchar(512) NOT NULL COMMENT 'person access token',
```

Comment:
token is a private ingo, just like password, we should encrypt it, just like what we do for connection config~

Reply:
Okay, I will encrypt it in service layer in the future.

Reply:
okay, you can take a look at the table `connect_connection`, there is several columns related to encryption, just like `cipher`, `salt`

Reply:
Thanks for your remind! I added `salt` into the entity but missed to add into ddl.